s2core/src/test/scala/com/kakao/s2graph/core/Integrate/IntegrateCommon.scala

```diff
@@ -113,48 +113,89 @@ trait IntegrateCommon extends FunSuite with Matchers with BeforeAndAfterAll {
     def getEdgesSync(queryJson: JsValue): JsValue = {
       logger.info(Json.prettyPrint(queryJson))
 
```

Comment:
Quick questions here. it seems that below code is duplicate. why not using Await.result(getEdgesAsync(jsQuery)(PostProcess.toSimpleVertexArrJson), waitTime) instead of duplicate code blocks?